tests/python/integration/test_scenario.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+from dku_plugin_test_utils import dss_scenario
+
+TEST_PROJECT_KEY = "PLUGINTESTDEEPLEARNINGIMAGE"
+
+
+def add_integration_test(user_dss_clients, scenario_id):
+    dss_scenario.run(user_dss_clients, project_key=TEST_PROJECT_KEY, scenario_id=scenario_id)
+
+
+def test_run_tensorboard_start(user_dss_clients):
+    add_integration_test(user_dss_clients, "Webapp_-_Tensorboard")
+
+
+def test_extract_recipe(user_dss_clients):
+    add_integration_test(user_dss_clients, "Recipe_-_Extract_features")
+
+
+def test_score_recipe(user_dss_clients):
+    add_integration_test(user_dss_clients, "Recipe_-_Classify_images")
+
+
+def test_retrain_recipe(user_dss_clients):
```

Comment:
- Same as above, print the missing files from the Filenames check

Reply:
Did so in my test project